### PR TITLE
Bump mssql connector

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Bump the connector version to make sure that the state lifecycle manager is included.

### CODE REVIEW

File: docs/integrations/destinations/mssql.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Using this feature requires additional configuration, when creating the source.
 
 | Version | Date | Pull Request                                             | Subject                                                                                             |
 |:--------| :--- |:---------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| 0.1.20  | 2022-07-12 | [14649](https://github.com/airbytehq/airbyte/pull/14649) | Include lifecycle management                                            |
```
Comment (subodh1810) [inline docs/integrations/destinations/mssql.md:118]: State Lifecycle management?

**Review by subodh1810 — APPROVED** [review]

File: docs/integrations/destinations/mssql.md (hunk 1 above)
Comment (benmoriceau) [reply docs/integrations/destinations/mssql.md:118]: It refers to https://github.com/airbytehq/airbyte/pull/13670.

**Comment by benmoriceau** [comment]
> 

No

> @benmoriceau have you tested this upgrade scenario:
> 
> 1. a full refresh is created with a connector that is not aware of state lifecycle manager (e.g. version before this one)
> 2. an incremental refresh is run with a connector that is not aware of state lifecycle manager
> 3. connector is upgraded to this version
> 4. an incremental sync runs and does not fail

@grishick No, I'm gonna try to do it.

**Comment by benmoriceau** [comment]
> > 
> 
> No
> 
> > @benmoriceau have you tested this upgrade scenario:
> > 
> > 1. a full refresh is created with a connector that is not aware of state lifecycle manager (e.g. version before this one)
> > 2. an incremental refresh is run with a connector that is not aware of state lifecycle manager
> > 3. connector is upgraded to this version
> > 4. an incremental sync runs and does not fail
> 
> @grishick No, I'm gonna try to do it.

@grishick I tested it locally and it worked, I'll publish once I have your approval.

**Review by grishick — APPROVED** [review]

**Pull request closed without merging**